optimade/server/config.py

```diff
@@ -87,8 +87,6 @@ def config_file_settings(settings: BaseSettings) -> Dict[str, Any]:
     import json
     import os
 
-    import yaml
```

Comment:
Is this still necessary?

Reply:
Yes and I do not see why the import of yaml needs to be moved back.
This way, no error is generated when yaml is not present. And it also slightly improves performance, as yaml does not need to be loaded when the settings are not in a yaml file.

You can see the error message I got with yaml in the original place here.
https://github.com/Materials-Consortia/optimade-python-tools/actions/runs/6392229085